### PR TITLE
styles: fix styling of headings and scroll to top

### LINKED ISSUE #761: [Ember-Migration] Refactor styling issues
### Issue Description

There are few styling refactors/fixes we can which are:
- Inconsistent font size of different section.
- Scroll to top button shows even if there is nothing to scroll.
- Old component showing on dev flag.

### Expected Behavior

There are expected behavior for above points:

- Same font size across different sections.
- Scroll to top button to show only when there is some height to scroll.
- Add old component to not show on dev flag.

### Current Behavior

There are few styling refactors/fixes we can which are:
- Inconsistent font size of different section.
- Scroll to top button shows even if there is nothing to scroll.
- Old component showing on dev flag.

### Screenshots

image
> Inconsistent font size of different section.
image
> Inconsistent font size of different section.
image
> Scroll to top button shows even if there is nothing to scroll.
image
> Old component showing on dev flag.

### Reproducibility

 - [x] This issue is reproducible
-  [ ] This issue is not reproducible

### Steps to Reproduce
1. [Step 1] -  Go to https://staging-www.realdevsquad.com/?dev=true
2. [Step 2] - See the font size with the help of devtools and other one's can be seen directly.

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [x] Low

### Additional Information

[Include any additional context, logs, or details that might be helpful in addressing the issue.]

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

---

Thank you for contributing to our project! We appreciate your help in improving it.

### PR DESCRIPTION
**Pull request by satyam73**
### What is the change?

These are the following changes that has been done:

- Added same font size across different sections.
- Fixed scroll to top button to show only when there is some height to scroll.
- Added old component to not show on dev flag.

##### Example :
- Added debounce on scroll effect to check if top=0 then don't show the scroll to top button.
- Added same stylings for different headings.
- Added conditional rendering to not show old component on dev=true

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots
image
image##### Example:
image
> No scroll to top button showing
image
> no old component showing on dev=true
image
> Scroll bar added for quicker navigation
Before :
Image/Video

After :
Image/Video

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: app/components/scroll-to-top.js [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+        if (window.scrollY !== 0) {
+          this.isScrollToTopVisible = true;
+        } else {
+          this.isScrollToTopVisible = false;
+        }
+      },
+      1000,
+    );
+
+    document.body.onscroll = onBodyScroll;
```
Comment (shubhamsigdar1) [inline app/components/scroll-to-top.js:27]:
```suggestion
    document.body.onscroll = onBodyScroll;
```
is above line required??

File: app/components/more-about.hbs [hunk 2]
```diff
@@ -1,40 +1,43 @@
...
-        src='assets/images/mobile-marketing.png'
-        alt='infoNoteImg'
-      />
-    </span>
-  </section>
-  <section class='video__section'>
-    <h1 data-test-video-title class='video__heading'>
+<section class="info-video">
+  {{! TODO - remove this after migration }}
+  {{#unless @isDev}}
```
Comment (shubhamsigdar1) [inline app/components/more-about.hbs:3]: NIT: Satyam I Didn't understand this part
Why you put on under dev flag, If it is already

File: app/components/scroll-to-top.js [hunk 3]
```diff
@@ -1,13 +1,49 @@
...
+    const onBodyScroll = myDebounce(
+      this,
+      function () {
+        if (window.scrollY !== 0) {
+          this.isScrollToTopVisible = true;
+        } else {
+          this.isScrollToTopVisible = false;
+        }
+      },
+      1000,
```
Comment (shubhamsigdar1) [inline app/components/scroll-to-top.js:24]: Nit: Can we decrease time to 500??

Comment (satyam73) [reply app/components/scroll-to-top.js:24]: Yep reduced

File: app/components/scroll-to-top.js (hunk 1 above)
Comment (satyam73) [reply app/components/scroll-to-top.js:27]: removed, forgot to remove

File: app/components/more-about.hbs (hunk 2 above)
Comment (satyam73) [reply app/components/more-about.hbs:3]: all the new changes are in dev=true so that's why did this

**Review by ajoykumardas12 — APPROVED** [review]

**Pull request merged**